src/macs/hmac.rs

```diff
@@ -6,26 +6,26 @@
 #[cfg(feature = "sha")]
 #[cfg_attr(docsrs, doc(cfg(feature = "sha")))]
 pub fn HMAC_SHA256(data: &[u8], key: &[u8], mac: &mut [u8; 32]) {
-    use hmac_::Mac;
-    let mut m = hmac_::Hmac::<sha2::Sha256>::new_from_slice(key).unwrap();
+    use hmac::Mac;
```

Comment:
I feel like we shouldn't use Mac trait here, it's going to change in the next version. Should probably use KeyInit and Update/Digest instead.

Reply:
`Mac` trait in [digest-0.11](https://docs.rs/digest/0.11.0-pre/digest/trait.Mac.html) will not have `new` method. Right now there's no need to prepare for that.